djangocms_versioning/admin.py

```diff
@@ -537,6 +537,22 @@ def _get_unpublish_link(self, obj, request, disabled=False):
             {"unpublish_url": unpublish_url, "disabled": disabled},
         )
 
+    def _get_published_link(self, obj, request):
+        """Helper function to get the link to the published page"""
+        try:
+            if not obj.state == PUBLISHED:
+                # Don't display the link if it isn't published
+                return ""
+
+            published_url = obj.content.get_absolute_url()
+        except AttributeError:
+            return ""
+
+        return render_to_string(
+            "djangocms_versioning/admin/published_icon.html",
```

Comment:
An icon for preview already exists here, we shouldn't need to create another one: https://github.com/django-cms/djangocms-versioning/blob/master/djangocms_versioning/templates/djangocms_versioning/admin/preview.html

Reply:
@Aiky30 If I were to use that template, then the text describing the icon would be "preview" but it's a link to view the published page. I'll update my template and move it for consistency, but I think an explicit template makes sense.